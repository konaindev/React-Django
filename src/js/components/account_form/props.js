export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
