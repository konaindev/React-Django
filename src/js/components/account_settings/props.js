const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
