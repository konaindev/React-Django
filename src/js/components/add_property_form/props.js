export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
