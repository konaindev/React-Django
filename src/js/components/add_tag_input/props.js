const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
