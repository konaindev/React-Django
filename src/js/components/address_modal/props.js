export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
