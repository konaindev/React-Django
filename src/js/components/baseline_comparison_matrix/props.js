import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
