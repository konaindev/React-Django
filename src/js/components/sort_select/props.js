export const props = {
  options: [
    { label: "Recently Viewed", value: "recently" },
    { label: "By Property Mgr.", value: "propertyMgr" },
    { label: "By Asset Owner", value: "assetOwner" },
    { label: "By Region", value: "region" },
    { label: "By City", value: "city" },
    { label: "By Fund", value: "fund" },
    { label: "By performance", value: "performance" }
  ]
};
