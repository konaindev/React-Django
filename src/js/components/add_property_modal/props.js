import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
