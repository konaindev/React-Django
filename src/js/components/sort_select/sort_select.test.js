import React from "react";
import renderer from "react-test-renderer";

import SortSelect from "./index";
import { props } from "./props";

describe("SortSelect", () => {
  it("render sort select", () => {
    const tree = renderer.create(<SortSelect {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("render reverse sort select", () => {
    const tree = renderer
      .create(<SortSelect {...props} direction="desc" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
