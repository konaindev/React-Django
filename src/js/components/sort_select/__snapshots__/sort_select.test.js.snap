// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortSelect render reverse sort select 1`] = `
<div
  className="sort-select"
>
  <div
    className="sort-select__title"
  >
    SORT:
  </div>
  <div
    className="sort-select__field-wrapper"
  >
    <div
      className="select sort-select__field css-2b097c-container"
      onKeyDown={[Function]}
    >
      <div
        className="select__control css-yk16xz-control"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <div
          className="select__value-container css-1hwfws3"
        >
          <div
            className="select__placeholder css-1wa3eu0-placeholder"
          >
            Select...
          </div>
          <input
            className="css-62g3xt-dummyInput"
            disabled={false}
            id="react-select-3-input"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            readOnly={true}
            tabIndex="0"
            value=""
          />
        </div>
        <div
          className="select__indicators css-1wy0on6"
        >
          <span
            className="select__indicator-separator css-1okebmr-indicatorSeparator"
          />
          <div
            aria-hidden="true"
            className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <svg
              aria-hidden="true"
              className="select__dropdown-arrow select__dropdown-arrow--down"
              color="inherit"
              focusable="false"
              height="1em"
              role="presentation"
              viewBox="0 0 10 5"
              width="1em"
            >
              <path
                d="M1 1.16134L4.92334 4.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
              <path
                d="M8.82812 1.16134L5 4.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
            </svg>
            <svg
              aria-hidden="true"
              className="select__dropdown-arrow select__dropdown-arrow--up"
              color="inherit"
              focusable="false"
              height="1em"
              role="presentation"
              viewBox="0 0 10 5"
              width="1em"
            >
              <path
                d="M1 3.83866L4.92334 0.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
              <path
                d="M8.82812 3.83866L5 0.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
    <svg
      aria-hidden="true"
      className="sort-select__icon sort-select__icon--reverse"
      color="inherit"
      focusable="false"
      height="1em"
      onClick={[Function]}
      role="presentation"
      viewBox="0 0 16 14"
      width="1em"
    >
      <g
        fill="currentColor"
        stroke="none"
        strokeWidth="1"
      >
        <g
          transform="translate(-1383.000000, -185.000000)"
        >
          <g
            transform="translate(1087.000000, 169.000000)"
          >
            <g
              transform="translate(296.000000, 16.000000)"
            >
              <rect
                height="1"
                rx="0.4895163"
                width="7.78365217"
                x="8.01085217"
                y="2.353442"
              />
              <rect
                height="1"
                rx="0.4895163"
                width="5.8377113"
                x="8.01085217"
                y="5.290502"
              />
              <rect
                height="1"
                rx="0.4895163"
                width="3.89181217"
                x="8.01085217"
                y="8.22766"
              />
              <rect
                height="1"
                rx="0.4895163"
                width="1.94590609"
                x="8.01085217"
                y="11.16472"
              />
              <path
                d="M2.85056,13.92804 C2.94554435,14.02394 3.09984696,14.02401 3.1949287,13.92818 L5.98018783,11.12216 C6.07507478,11.02654 6.07503304,10.87163 5.98008348,10.77608 L5.68895304,10.48313 C5.59389913,10.38751 5.43977043,10.38758 5.34480696,10.48334 L3.47919304,12.3641 L3.47919304,0.2447585 C3.47919304,0.1095822 3.37029565,-1.139453e-08 3.23595826,0 L2.80937043,3.618335e-08 C2.67503304,4.757781e-08 2.56613565,0.1095822 2.56613565,0.2447585 L2.5661287,12.3641 L0.706685218,10.48383 C0.611766261,10.38786 0.457431652,10.38765 0.36227687,10.48341 L0.0712431306,10.77629 C-0.0236404172,10.87177 -0.0237631303,11.02654 0.070968348,11.12216 L2.85056,13.92804 Z"
              />
            </g>
          </g>
        </g>
      </g>
    </svg>
  </div>
</div>
`;

exports[`SortSelect render sort select 1`] = `
<div
  className="sort-select"
>
  <div
    className="sort-select__title"
  >
    SORT:
  </div>
  <div
    className="sort-select__field-wrapper"
  >
    <div
      className="select sort-select__field css-2b097c-container"
      onKeyDown={[Function]}
    >
      <div
        className="select__control css-yk16xz-control"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <div
          className="select__value-container css-1hwfws3"
        >
          <div
            className="select__placeholder css-1wa3eu0-placeholder"
          >
            Select...
          </div>
          <input
            className="css-62g3xt-dummyInput"
            disabled={false}
            id="react-select-2-input"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            readOnly={true}
            tabIndex="0"
            value=""
          />
        </div>
        <div
          className="select__indicators css-1wy0on6"
        >
          <span
            className="select__indicator-separator css-1okebmr-indicatorSeparator"
          />
          <div
            aria-hidden="true"
            className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <svg
              aria-hidden="true"
              className="select__dropdown-arrow select__dropdown-arrow--down"
              color="inherit"
              focusable="false"
              height="1em"
              role="presentation"
              viewBox="0 0 10 5"
              width="1em"
            >
              <path
                d="M1 1.16134L4.92334 4.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
              <path
                d="M8.82812 1.16134L5 4.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
            </svg>
            <svg
              aria-hidden="true"
              className="select__dropdown-arrow select__dropdown-arrow--up"
              color="inherit"
              focusable="false"
              height="1em"
              role="presentation"
              viewBox="0 0 10 5"
              width="1em"
            >
              <path
                d="M1 3.83866L4.92334 0.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
              <path
                d="M8.82812 3.83866L5 0.5"
                stroke="currentColor"
                strokeLinecap="round"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
    <svg
      aria-hidden="true"
      className="sort-select__icon"
      color="inherit"
      focusable="false"
      height="1em"
      onClick={[Function]}
      role="presentation"
      viewBox="0 0 16 14"
      width="1em"
    >
      <g
        fill="currentColor"
        stroke="none"
        strokeWidth="1"
      >
        <g
          transform="translate(-1383.000000, -185.000000)"
        >
          <g
            transform="translate(1087.000000, 169.000000)"
          >
            <g
              transform="translate(296.000000, 16.000000)"
            >
              <rect
                height="1"
                rx="0.4895163"
                width="7.78365217"
                x="8.01085217"
                y="2.353442"
              />
              <rect
                height="1"
                rx="0.4895163"
                width="5.8377113"
                x="8.01085217"
                y="5.290502"
              />
              <rect
                height="1"
                rx="0.4895163"
                width="3.89181217"
                x="8.01085217"
                y="8.22766"
              />
              <rect
                height="1"
                rx="0.4895163"
                width="1.94590609"
                x="8.01085217"
                y="11.16472"
              />
              <path
                d="M2.85056,13.92804 C2.94554435,14.02394 3.09984696,14.02401 3.1949287,13.92818 L5.98018783,11.12216 C6.07507478,11.02654 6.07503304,10.87163 5.98008348,10.77608 L5.68895304,10.48313 C5.59389913,10.38751 5.43977043,10.38758 5.34480696,10.48334 L3.47919304,12.3641 L3.47919304,0.2447585 C3.47919304,0.1095822 3.37029565,-1.139453e-08 3.23595826,0 L2.80937043,3.618335e-08 C2.67503304,4.757781e-08 2.56613565,0.1095822 2.56613565,0.2447585 L2.5661287,12.3641 L0.706685218,10.48383 C0.611766261,10.38786 0.457431652,10.38765 0.36227687,10.48341 L0.0712431306,10.77629 C-0.0236404172,10.87177 -0.0237631303,11.02654 0.070968348,11.12216 L2.85056,13.92804 Z"
              />
            </g>
          </g>
        </g>
      </g>
    </svg>
  </div>
</div>
`;
